src/pages/Admin/Charity/Donations/DonationsTable.tsx

```diff
@@ -27,15 +30,64 @@ export default function DonationsTable({ classes = "" }) {
         }}
       >
         {(donations) => (
-          <Table
-            donations={donations}
-            hasMore={hasMore}
-            onLoadMore={loadNextPage}
-            disabled={isLoadingOrError}
-            isLoading={isLoadingNextPage}
-          />
+          <>
+            <div className="grid w-full sm:flex items-center sm:justify-end mb-2 gap-2">
+              <CsvExporter
+                classes="border border-blue text-blue-d1 hover:border-blue-l2 hover:text-blue rounded px-4 py-2 text-sm"
+                headers={csvHeadersDonations}
+                data={donations}
+                filename="received_donations.csv"
+              >
+                <Icon type="FileCSV" size={17} className="text-2xl" />
+                Donation Records
+              </CsvExporter>
+              <CsvExporter
+                classes="border border-blue text-blue-d1 hover:border-blue-l2 hover:text-blue rounded px-4 py-2 text-sm"
+                headers={csvHeadersReceipts}
+                data={donations
+                  .filter((x) => !!x.kycData)
+                  .map((x) => x.kycData!)}
```

Comment:
small tip: to avoid having to assert existence with `x.kycData!`, you can make use of `filter`'s ability to do the same:
```ts
  data={donations
    .map((x) => x.kycData)
    .filter((kycData): kycData is KYCData => !!kycData)} // TS now knows that `kycData` is defined
```